org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/preferences/BuilderPreferenceAccess.java

```diff
@@ -32,6 +33,37 @@
 	 * Name of a preference for configuring whether the builder participant is enabled or not.
 	 */
 	public static final String PREF_AUTO_BUILDING = "autobuilding"; //$NON-NLS-1$
+	/**
+	 * Name of a preference defining the scheduling rule strategy for the builder.
+	 * @since 2.16
+	 */
+	@Beta
+	public static final String PREF_SCHEDULING_RULE = "schedulingrule"; //$NON-NLS-1$
+	/**
+	 * Options for scheduling rules.
+	 * @since 2.16
+	 */
+	public static enum SchedulingOption {
+		WORKSPACE ("workspace"),
+		// all Xtext projects in the workspace
+		XTEXT_PROJECTS ("all_xtext_projects"), //$NON-NLS-1$
+		// current built project
```

Comment:
Built = past tense, what exactly does "current built project" mean?
I think what you meant to say was "The currently building project"

Reply:
This is indeed better